test/index.js

```diff
@@ -407,13 +408,6 @@ describe('DDWAF', () => {
     const possibleValues = new Map([
       [undefined, undefined],
       [null, undefined],
-      [false, 'false'],
-      [true, 'true'],
-      [42, '42'],
-      [-42, '-42'],
-      [42.42, '42.42'],
```

Comment:
I can understand removing Infinity and NaN because they're not really useful values, but why not keep the test for bools and numbers ?

Reply:
I guess those test were placed in order to test that a number will be coerced to string. Since we're not doing that anymore those tests seemed kind of redundant. Anyway if you really think they are really useful we can put them back.

Reply:
I guess there is no point to support these kind of values for matching rules. However I will test with all kind of types in the derivative tests.